hphp/runtime/vm/native.h

```diff
@@ -231,6 +231,7 @@ bool coerceFCallArgs(TypedValue* args,
  * calling.
  */
 template<bool usesDoubles>
+__attribute__((noinline))
```

Comment:
Why is this changing? I would have thought changing how we inline `callFunc` could impact performance.